logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -17,6 +17,7 @@ function Get-DefaultDevelopmentSettingsByProject {
             }
             ConnectionStrings = @{ }
             Plugin            = @{
+                Folder = "../../Plugin"
```

Comment:
So what should the default Folder be for the appsettings.json?

Reply:
good question, i need this to make the populated template to work. but I was unsure if I should change production version of the link in the appsettings.json.

Reply:
This folder is moving anyways into WebApi, so this is fine for now.